Rakefile

```diff
@@ -1,10 +1,10 @@
-require "bundler/gem_tasks"
-require "rake/testtask"
+require 'bundler/gem_tasks'
+require 'rake/testtask'
```

Comment:
Are you adopting a convention of only using double quotes when needed? Just curious.

Reply:
I obeyed the default settings for the Style/StringLiterals rubocop here, which made me change all of these to `'`.

That experience is why I rejected that cop in redis-key_hash redis-script_manager.